spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -101,3 +101,12 @@ fun SpdxLicense.toExpression(): SpdxLicenseIdExpression {
  */
 fun String.toSpdx(strictness: Strictness = Strictness.ALLOW_ANY): SpdxExpression =
     SpdxExpression.parse(this, strictness)
+
+/**
+ * Aggregates multiple SpdxExpressions to a [SpdxCompoundExpression] concatenated using [SpdxOperator.OR].
```

Comment:
This should use the imperative and say "Aggregate" (I just realized it's also wrong in the existing function docs above).